### PR TITLE
Update of french translation

### PR DESCRIPTION
**Pull request by Azelout**
Hey !
I updated the french translation.

You can check the preview right below ! :)

player's actions
player's historic
player's infos
txAdmin menu
announcement
ban player
player banned
player

### CODE REVIEW

File: locale/fr.json [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
     },
     "ban_messages": {
-        "kick_temporary": "(%{author}) You have been banned from this server for \"%{reason}\". Your ban will expire in: %{expiration}.",
-        "kick_permanent": "(%{author}) You have been permanently banned from this server for \"%{reason}\".",
-        "reject_temporary": "You have been banned from this server.\nYour ban will expire in: %{expiration}.\nBan ID: %{id}.\nBan Reason: %{reason}.\nBanned by: %{author}.",
-        "reject_permanent": "You have been permanently banned from this server.\nBan ID: %{id}.\nBan Reason: %{reason}.\nBanned by: %{author}."
+        "kick_temporary": "(%{author}) Vous avez été banni pour: \"%{reason}\". Votre bannissement expirera dans: %{expiration}.",
+        "kick_permanent": "(%{author}) Vous avez été définitivement banni de ce serveur pour: \"%{reason}\".",
+        "reject_temporary": "Vous avez été banni de ce serveur.\nVotre bannissement expirera dans: %{expiration}.\nBan ID: %{id}.\nRaison du Ban: %{reason}.\nBanni par: %{author}.",
+        "reject_permanent": "Vous avez été définitivement banni de ce serveur pour.\nBan ID: %{id}.\nRaison du Ban: %{reason}.\nBanni par: %{author}."
```
Comment (keketiger) [inline locale/fr.json]: The word (`pour`) must be removed. On line 20.

Comment (Azelout) [reply locale/fr.json]: I'll make the change tonight.
Thank you

File: locale/fr.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
         "spawning_discord": "**%{servername}** est en train de démarrer."
     },
     "discord": {
-        "status_online": "**%{servername}** est actuellement **Ouvert**!",
-        "status_offline": "**%{servername}** est actuellement **Fermer**!"
+        "status_online": "**%{servername}** est actuellement **Ouvert** !",
+        "status_offline": "**%{servername}** est actuellement **Fermer** !"
```
Comment (tabarra) [inline locale/fr.json]: Why this added space?
Someone on discord just told me:
> its not "est actuellement Fermer"
> its "est actuellement Fermé"

Comment (keketiger) [reply locale/fr.json]: Yes is "Fermé" not "Fermer".

Comment (Azelout) [reply locale/fr.json]: I corrected that thanks
I added spaces because in French we have space before an exclamation mark and a question mark

**Comment by tabarra** [comment]
> In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation) should always have a space preceding them.

Huh, I didn't know that.

**Comment by tabarra** [comment]
Thanks for the PR.  
These options below are too big, it should be two lines only.  
Can you please be kind and make a new PR shortening those options?
image

**Pull request merged**